pkgs/tools/admin/qovery-cli/default.nix

```diff
@@ -8,16 +8,16 @@
 
 buildGoModule rec {
   pname = "qovery-cli";
-  version = "0.61.0";
+  version = "0.63.0";
 
   src = fetchFromGitHub {
     owner = "Qovery";
     repo = pname;
-    rev = "v${version}";
-    hash = "sha256-V3tKUIpSO1oGpkHjkFWAtBoe91va6q2HInZsHrjWhQ0=";
+    rev = "refs/tagsv${version}";
```

Comment:
missing slash, so that Hydra fails:
https://hydra.nixos.org/log/gscsd5c3277iv9cvz5mszw3np6qkmvds-source.drv
(you probably had it prefetched locally)